backend/fms_core/utils.py

```diff
@@ -88,6 +88,13 @@ def get_normalized_str(d: dict, key: str, default: str = "") -> str:
     """
     return str_cast_and_normalize(d.get(key) or default)
 
+
+def is_date_after_today(date):
+    if not isinstance(date, datetime.date):
```

Comment:
Should this also support `datetime` objects, and use `date.date()` to convert to a simple date?